shared/src/utils/validators/is-string-literal-union.ts

```diff
@@ -0,0 +1,18 @@
+import type { ValidationOptions, ValidationArguments } from 'class-validator';
+import { isString } from 'class-validator';
+import { makeValidator } from './make-validator';
+
+// TODO: This should actually verify the type
```

Comment:
I was already getting my hopes up XD

Why do you want to change it from `isString` to this?

Reply:
This allows me to find them more easily when I actually implement proper validation for them. And until then they at least convey the intent to the reader (unfortunately not to the runtime, though).